frameit/frames_generator/Rakefile

```diff
@@ -0,0 +1,75 @@
+require 'colored'
+require 'mini_magick'
+require 'json'
+
+DEVICES_TO_SKIP = [
+  "Apple iMac Retina.png", # we don't currently support iMac
+  "Apple iMac.png" # we don't currently support iMac
+]
+
+task :generate_device_frames do
+  MiniMagick.configure do |config|
+    config.validate_on_create = false
+  end
+
+  puts "Download and extract the latest devices from Facebook"
+  puts ""
+  puts "  http://facebook.design/devices"
+  puts ""
+  confirm
+  puts "Move the content of the zip file into './raw'"
+  mkdir("raw") unless File.directory?("raw")
+  system("open ./raw")
+  confirm
+
+  # Let's find all the image resources we need and copy it into the correct directory
+  current_time = Time.now.to_i.to_s
+  output = File.join("output", current_time)
```

Comment:
I like a directory name that sorts well and has some meaning. Maybe append a hash to the end?